packages/oas-to-har/src/index.js

```diff
@@ -222,8 +222,14 @@ module.exports = (
     });
   }
 
-  const schema = getSchema(operation.schema, oas) || { schema: {} };
-  if (schema.schema && Object.keys(schema.schema).length) {
+  let requestBody = getSchema(operation.schema, oas);
```

Comment:
Made this change because the API for `getSchema` slightly changed in https://github.com/readmeio/oas/pull/358.